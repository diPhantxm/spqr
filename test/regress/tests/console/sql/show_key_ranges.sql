DROP KEY RANGE krid4;
DROP KEY RANGE krid5;
SHOW key_ranges;