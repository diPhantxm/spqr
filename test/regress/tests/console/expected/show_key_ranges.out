
		SQPR router admin console
	Here you can configure your routing rules
------------------------------------------------
	You can find documentation here 
https://github.com/pg-sharding/spqr/tree/master/docs

SHOW key_ranges;
 Key range ID | Shard ID |             Lower bound              |             Upper bound              
--------------+----------+--------------------------------------+--------------------------------------
 krid5        | sh2      | 80000000-0000-0000-0000-000000000001 | FFFFFFFF-FFFF-FFFF-FFFF-FFFFFFFFFFFF
 krid4        | sh1      | 00000000-0000-0000-0000-000000000000 | 80000000-0000-0000-0000-000000000000
 krid3        | sh2      | 21                                   | 30
 krid2        | sh1      | 11                                   | 20
 krid1        | sh1      | 1                                    | 10
(5 rows)

